streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/register/impl/AbstractServiceRegistrar.java

```diff
@@ -117,8 +126,26 @@ public Service register(Cluster cluster, Config config, List<ConfigFileInfo> con
         return service;
     }
 
+    protected boolean validateComponentProcesses(List<ComponentProcess> componentProcesses) {
```

Comment:
plural form and List look like duplicated. `isComponentProcessesValid` looks OK.